planner/languages/rust/rust_planner.go

```diff
@@ -17,9 +29,87 @@ func (p *Planner) Name() string {
 }
 
 func (p *Planner) IsRelevant(srcDir string) bool {
-	return false
+	return p.cargoTomlPath(srcDir) != ""
 }
 
 func (p *Planner) GetPlan(srcDir string) *plansdk.Plan {
-	return &plansdk.Plan{}
+	plan, err := p.getPlan(srcDir)
+	if err != nil {
+		if plan == nil {
+			plan = &plansdk.Plan{}
+		}
+		plan.WithError(err)
+	}
+	return plan
+}
+
+func (p *Planner) getPlan(srcDir string) (*plansdk.Plan, error) {
+
+	rustVersion, err := p.rustOxalicaVersion(srcDir)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	rustPkgDev := fmt.Sprintf("rust-bin.stable.%s.default", rustVersion)
+	rustPkgRuntime := fmt.Sprintf("rust-bin.stable.%s.minimal", rustVersion)
+
+	return &plansdk.Plan{
+		Overlays:        []string{RustOxalicaOverlay},
+		DevPackages:     []string{rustPkgDev},
+		RuntimePackages: []string{rustPkgRuntime},
```

Comment:
This RuntimePackages should be part of the next PR. I can drop it on my next update.

 I'll also look into whether we can avoid it entirely, similar to the go planner not needing any runtime packages.